detection-rules/credential_phishing_docusign_embedded_image_lure.yml

```diff
@@ -5,7 +5,7 @@ severity: "high"
 source: |
   type.inbound
   and length(attachments) <= 1
-  and any(body.links,
+  and all(body.links,
```

Comment:
### 
```suggestion
  and length(body.links) > 0
  and all(body.links,
```

Adding a length check to precede the `all(body.links` check to prevent a false truthin the event the array is empty 


all returns true if all of the values in the original array match an expression, which is provided by the second argument. If the array is empty, then all is vacuously `true. [Reference](https://docs.sublimesecurity.com/docs/functions#all)